types/index.d.ts

```diff
@@ -820,6 +820,11 @@ declare namespace RamdaAdjunct {
         flattenProp(prop: string | number, obj: object): object;
         flattenProp(prop: string | number): (obj: object) => object;
 
+        /**
+         * Creates a new list out of the supplied object by applying the function to each key/value pairing.
+         */
+        unzipObjWith<T, U, V>(fn: (value: T, key: string) => [U, V], obj: {}): [U[], V[]];
```

Comment:
Can we align signatures to look like this ?
```ts
unzipObjWith<T, U>(fn: (v: T, k: string) => [U, T], obj: { [k: string]: T }): [U[], T[]];
unzipObjWith<T, U>(fn: (v: T, k: string) => [U, T]): (obj: { [k: string]: T }) => [U[], T[]];
```
Notice the curried form of the signature.